packages/styles/src/elements/heading.scss

```
@@ -0,0 +1,62 @@
+@use '../functions/tokens' as tokens;
+@use '../tokens/elements' as elements;
```

With `@use`, the file name automatically becomes the namespace. Overriding the namespace name with `as X` is only necessary when the filename is not optimal or two identically named files are being imported.

```suggestion
@use '../functions/tokens';
@use '../tokens/elements';
```